crates/polars-stream/src/physical_plan/lower_ir.rs

```diff
@@ -26,7 +26,7 @@ pub fn lower_ir(
     let output_schema = IR::schema_with_cache(node, ir_arena, schema_cache);
     let node_kind = match ir_node {
         IR::SimpleProjection { input, columns } => {
-            let columns = columns.get_names_owned();
+            let columns = columns.iter_names().cloned().collect::<Vec<_>>();
```

Comment:
Given that we call this often, a utility method might save some code bloat. :eyes: